sdk/ai/azure-ai-projects/samples/evaluations/sample_evaluations.py

```diff
@@ -82,5 +82,6 @@
 print("----------------------------------------------------------------")
 print("Created evaluation, evaluation ID: ", get_evaluation_response.id)
 print("Evaluation status: ", get_evaluation_response.status)
-print("AI Studio URI: ", get_evaluation_response.properties["AiStudioEvaluationUri"])
+if isinstance(get_evaluation_response.properties, dict):
+    print("AI Studio URI: ", get_evaluation_response.properties["AiStudioEvaluationUri"])
```

Comment:
Brand can/will change, can we avoid using AI Studio? E,g, in property value? AiStudioEvaluationUri, also in prints.

Reply:
@singankit also to comment from evaluation side